src/component/Timer/index.jsx

```diff
@@ -1,9 +1,18 @@
+/* eslint-disable space-before-blocks */
+/* eslint-disable space-before-function-paren */
+/* eslint-disable no-undef */
+/* eslint-disable vars-on-top */
+/* eslint-disable no-var */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable indent */
+/* eslint-disable comma-dangle */
```

Comment:
You should remove all of these eslint disable comments. We don't want to disable these rules. It's OK to disable rules while developing code to stay fast, but you should not commit these to Git.

Reply:
yes, absolutely I shouldn't push these eslient disable comments. sorry I forget them.